cascalog-core/test/cascalog/cascading/operations_test.clj

```diff
@@ -19,7 +19,7 @@
 (defn produces
   ([gen] (produces gen :fatal))
   ([gen log-level]
-     (chatty-checker
```

Comment:
What outputs are you getting on failing tests with this change? chatty-checker was the only thing giving us some visibility into what's happening with the outputs, yeah?